lib/bundler/settings.rb

```diff
@@ -176,11 +185,14 @@ def groups_array(group_type, scope)
     # Always returns an absolute path to the bundle directory
     # TODO: Refactor this method
     def path
+      # TODO: Shoudl we consider the @current_config here?
```

Comment:
Never mind -- I don't think we need to. The comment just below it explains why: `--path` is no longer used, so `bundle install --path foo` or any command that previously accepted `--path` would now be set with `bundle config`, which only set options in the `local` and `global` scopes.